### ISSUE TITLE
Mobile view of Qualtrics request forms - no way to get back to Kits

### ISSUE THREAD

**Issue by swwalker** [DukeLearningInnovation/kits#324]
Once user enters text into the form, there is no way to get back to the app.  This was the response when using the back button on the form:

![image](https://user-images.githubusercontent.com/38253046/49315431-ae971a00-f4bb-11e8-9dc3-7860b1fe9d68.png)

This was observed when accessing the Help form from the hamburger menu drawer.

**Comment by bap13** [comment]
Tested 12/17 on iphone 6SE in Safari, and chrome on a desktop machine: there is now a button to get back to kits from two qualtrics forms I found:
1. Help form accessed from mobile menu
2. Link on About page to request a new app be  added to Kits

The buttons take me back to kits from Qualtrics, and though I am still logged in, I do not see any of my classes. Screen grabs attached:

![img_7403](https://user-images.githubusercontent.com/38254766/50093280-3b411800-01de-11e9-9acf-27fc84a88bae.png)
![screen shot 2018-12-17 at 9 28 15 am](https://user-images.githubusercontent.com/38254766/50093289-3ed49f00-01de-11e9-8141-5a63bd6d3f90.png)
